src/pages/migrating/guide/overview.mdx

```diff
@@ -0,0 +1,216 @@
+---
+title: Guide
+description:
+  This guide includes everything you need to migrate your offering from an older
+  version of Carbon to v11.
+tabs: ['Overview', 'Design', 'Develop']
+---
+
+<PageDescription>
+
+The Carbon v11 release includes quality of life updates for designers and
+developers, and with no brand updates, no product redesigns are required.
+
+</PageDescription>
+
+<InlineNotification>
+
+**Note:** The Carbon v11 release is currently in beta release. Be on the lookout
+for the public preview release available in the coming months.
+
+</InlineNotification>
+
+<AnchorLinks>
+
+<AnchorLink>Overview</AnchorLink>
+<AnchorLink>Read</AnchorLink>
+<AnchorLink>Watch</AnchorLink>
+<AnchorLink>Release FAQs</AnchorLink>
+
+</AnchorLinks>
+
+## Overview
+
+Carbon v11 focuses on quality of life updates for designers and developers
+including token and prop renaming for ease of use, component API updates for a
+more predictable developer experience, and introducing CSS grid for accurate
+layout building in line with the design language.
+
+With no changes to the IBM Design Language, v11 will not require any
+brand-driven product redesigns.
+
+### What’s new
+
+- CSS grid lets you more accurately build layouts in line with the design
+  language.
+- Inline theming for light/dark modes.
+- Token renaming for better understanding and ease of use.
+- Size props introduced for icons to reduce icon package size.
+- Reduced compile times with new Sass modules.
```

Comment:
Maybe another way of putting this:

- Theming: light/dark mode now available along with design token updates
- Design Kits: TBD
- Packages: a new one-package install is available through `@carbon/react`
- Grid: CSS Grid support has landed to make it easier to build layouts
- React: new one-package install along with new components and API updates

Might help to make these more feature-oriented in terms of the end-user area instead of the feature set from the maintainer perspective.